web/packages/teleport/src/Console/DocumentKubeExec/DocumentKubeExec.story.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+
+import { createTeleportContext } from 'teleport/mocks/contexts';
+import ConsoleCtx from 'teleport/Console/consoleContext';
+
+import { ContextProvider } from 'teleport';
+import { TestLayout } from 'teleport/Console/Console.story';
+import TeleportContext from 'teleport/teleportContext';
+import * as stores from 'teleport/Console/stores/types';
+
+import DocumentKubeExec from './DocumentKubeExec';
+
+import type { Session } from 'teleport/services/session';
+
+export default {
+  title: 'Teleport/Console/DocumentKubeExec',
+};
+
+export const Connected = () => {
+  const { ctx, consoleCtx } = getContexts();
+
+  return (
+    <DocumentKubeExecWrapper ctx={ctx} consoleCtx={consoleCtx} doc={baseDoc} />
+  );
+};
+
+export const NotFound = () => {
+  const { ctx, consoleCtx } = getContexts();
+
+  const disconnectedDoc = {
+    ...baseDoc,
+    status: 'disconnected' as const,
+  };
+
+  return (
+    <DocumentKubeExecWrapper
+      ctx={ctx}
+      consoleCtx={consoleCtx}
+      doc={disconnectedDoc}
+    />
+  );
+};
+
+export const ServerError = () => {
+  const { ctx, consoleCtx } = getContexts();
+
+  const noSidDoc = {
+    ...baseDoc,
+    sid: '',
+  };
+
+  return (
+    <DocumentKubeExecWrapper ctx={ctx} consoleCtx={consoleCtx} doc={noSidDoc} />
+  );
+};
```

Comment:
In storybook, these three all render as the same to me, I'm not sure if that's what's supposed to happen.

I wanted to compare this against `DocumentSsh` stories, but it looks like stories for that component have been broken for some time.

Reply:
Yeah, `DocumentSsh` was broken, I've fixed it (PR fixing it: https://github.com/gravitational/teleport/pull/41877 ) and followed same pattern, it also renders basically same 🤷